app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/miband/MiBand2SampleProvider.java

```diff
@@ -47,13 +47,23 @@
     // 12 = definitely wake up
     // 17 = definitely not sleep related
 
+    // Firmware >1.0.1.47 changed sleep detection
+    // Those activities are marked wit NEWFW
```

Comment:
\*with

but I agree, should use the actual number, since a new firmware can pop up and then its not `NEWFW` but `MIDDLEOLDFW` ;)